aws/resource_aws_autoscaling_group.go

```diff
@@ -911,6 +921,12 @@ func resourceAwsAutoscalingGroupUpdate(d *schema.ResourceData, meta interface{})
 		opts.DefaultCooldown = aws.Int64(int64(d.Get("default_cooldown").(int)))
 	}
 
+	if d.HasChange("capacity_rebalance") {
+		if v, ok := d.GetOk("capacity_rebalance"); ok {
```

Comment:
This should be set even if `ok==false` otherwise going from `true` to removing the attribute will leave it `true`.

Reply:
Understood. Will submit change shortly.

Reply:
Addressed in commit bb483e8